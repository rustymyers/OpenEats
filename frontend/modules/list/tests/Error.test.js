import React from 'react';
import Error from '../components/Error';
import createComponentWithIntl from '../../../jest_mocks/createComponentWithIntl.js';

test('MyLists component test', () => {
  const component = createComponentWithIntl(
    <Error message="There is an error!"/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
