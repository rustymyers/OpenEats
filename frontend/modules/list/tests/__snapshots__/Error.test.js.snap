// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyLists component test 1`] = `
<div
  className="list-error"
>
  <div
    className="alert alert-danger"
    role="alert"
  >
    <span
      className="glyphicon glyphicon-exclamation-sign"
    />
    Â 
    <span>
      Something went wrong!
      : 
    </span>
    There is an error!
  </div>
</div>
`;
